doc/nrf/config_and_build/config_and_build_system.rst

```diff
@@ -77,6 +77,7 @@ The |NCS| build system places output images in the :file:`<build folder>/zephyr`
 These images are then used for updates from a cloud served (for example, nRF Cloud).
 
 If :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` is set, the build system creates the :file:`dfu_application.zip` file containing files :file:`app_update.bin` and :file:`manifest.json`.
+If you have set the :kconfig:option:`CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT`, the build system creates secondary slot variant of the :file:`app_update.bin` named :file:`mcuboot_secondary_app_update.bin` and includes it in the :file:`dfu_application.zip` file.
```

Comment:
I would write a follow-up sentence that
This configuration option and the direct-xip mode are only intended for singe core devices like the nRF52 series.